packages/metadata/src/Decorated/consts/fromMetadata/index.ts

```diff
@@ -23,19 +21,10 @@ export default function fromMetadata (registry: Registry, metadata: Metadata): C
 
     // For access, we change the index names, i.e. Democracy.EnactmentPeriod -> democracy.enactmentPeriod
     result[stringCamelCase(name.toString())] = moduleMetadata.constants.reduce((newModule: ModuleConstants, meta): ModuleConstants => {
-      // in the case of Bytes, the data has a length prefix encoded when received,
-      // leading to double-encoding unless removed
+      // convert to the natural type as received
       const type = meta.type.toString();
-      const codec: ConstantCodec = createTypeUnsafe(registry, type, [
-        AS_STRIPPED.includes(type)
-          ? meta.value.toU8a(true)
-          : meta.value
-      ]);
-
-      // This is not a perfect idea, however as it stands with number-only constants on the metadata
-      // does not have any effect. However, this could become problematic in cases where items are
-      // exposed that contain their own metadata. As of now, the compatibility with current, e.g.
-      // storage is the driving factor, one consistent way of handling interfaces
+      const codec: ConstantCodec = createTypeUnsafe(registry, type, [hexToU8a(meta.value.toHex())]);
```

Comment:
Remove the workaround we had, now -

- go to hex (which doesn't have the prefix
- convert that to the raw Uint8Array (which may have an _embedded_ prefix)

We cannot use Raw here, since that would assume no prefix added, so we need to go to a pure Uint8Array